src/erliam_xml.erl

```diff
@@ -12,48 +12,41 @@
 
 -include_lib("xmerl/include/xmerl.hrl").
 
-
 %%%% API
 
 parse(Xml) ->
     {E, _} = xmerl_scan:string(unicode:characters_to_list(Xml, utf8)),
     parse_element(E).
 
-
 %%%% INTERNAL FUNCTIONS
 
 parse_element(#xmlElement{name = Name, content = Content}) ->
-    {Name, case Content of
-               [#xmlElement{} | _] ->
-                   [{ChildName, parse_element(ChildContent)}
-                    || #xmlElement{name = ChildName,
-                                   content = ChildContent} <- Content];
-               _ ->
-                   parse_element(Content)
-           end};
-
+    {Name,
+     case Content of
+       [#xmlElement{} | _] ->
+           [{ChildName, parse_element(ChildContent)}
+            || #xmlElement{name = ChildName, content = ChildContent} <- Content];
+       _ ->
+           parse_element(Content)
+     end};
 parse_element([#xmlText{} | _] = L) ->
     case lists:all(fun (E) ->
-                           is_record(E, 'xmlText')
-                   end, L) of
-        true ->
-            unescape_xml_text(xmerl_xs:value_of(L));
-
-        %% if a list of elements is not solely text, discard the text elements and parse
-        %% the non-text elements:
-        false ->
-            parse_element([X
-                           || X <- L,
-                              not is_record(X, 'xmlText')])
+                           is_record(E, xmlText)
+                   end,
+                   L)
+        of
+      true ->
+          unescape_xml_text(xmerl_xs:value_of(L));
+      %% if a list of elements is not solely text, discard the text elements and parse
```

Comment:
It feels weird that a blank space is not left for the last statement and this new statement, maybe it because of that comment block in the middle

Reply:
Sadly… this is a bug that was not fixed with AdRoll/rebar3_format#95. I'm reopening AdRoll/rebar3_format#83 😞